401790a

commit 401790a
Author: Athanatos123 <alain.delree@gmail.com>
Date:   Tue Aug 25 09:09:57 2026 +0200

    Retire du .gitignore les règles des documents de travail déménagés hors dépôt (issue #251)
    
    TRAVAIL_EN_COURS/, TACHES.md, REARCHITECTURE_ALAIN.md, REARCHITECTURE_CLAUDE_CODE.md,
    HANDOFF_*.md, AUDIT_ALCHESS.md, Copier-coller, mockup_menu.html, test_perm.txt +
    commentaires devenus orphelins. logs/ conservé inchangé.

diff --git a/.gitignore b/.gitignore
index bb7eef4..7f575cd 100755
--- a/.gitignore
+++ b/.gitignore
@@ -11,13 +11,6 @@ build/
 # Parties temporaires
 games/tmp/*.pgn
 
-TRAVAIL_EN_COURS/
-TACHES.md
-
-#Fichiers rearchitecture Linux Windows Android
-REARCHITECTURE_ALAIN.md
-REARCHITECTURE_CLAUDE_CODE.md
-
 # Exécutables moteurs Windows (trop lourds, téléchargés par install_alchess.ps1)
 engines/stockfish*.exe
 engines/stockfish-windows*/
@@ -32,12 +25,6 @@ alchess_log.txt
 # --- Données personnelles (ne jamais publier) ---
 games/
 
-# --- Fichiers de travail / documents de handoff ---
-HANDOFF_*.md
-AUDIT_ALCHESS.md
-Copier-coller
-mockup_menu.html
-test_perm.txt
 logs/
 
 .assetsignore
